notanorm/mysql.py

```diff
@@ -1,15 +1,37 @@
+from collections import defaultdict
+
 import MySQLdb
 import MySQLdb.cursors
 
 from .base import DbBase
+from .model import DbType, DbModel, DbTable, DbCol, DbIndex
+from . import errors as err
+import re
 
+import logging as log
 # driver for mysql
 
 
 class MySqlDb(DbBase):
     placeholder = "%s"
-    retry_errors = (MySQLdb.OperationalError, )
-    integrity_errors = (MySQLdb.IntegrityError, )
+
+    @staticmethod
+    def translate_error(exp):
```

Comment:
No change needed, just a thought: if it winds up being useful to see the full orig exception, we could instead have this reraise exceptions `from exp`